dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/support/AbstractRegistryTest.java

```diff
@@ -195,4 +245,253 @@ public void notify2Test() {
         abstractRegistry.notify(testUrl, listener, urls);
         Assert.assertTrue(notifySuccess);
     }
+
+
+
+    /**
+     * test register
+     */
+    @Test
+    public void testRegister() {
```

Comment:
pls. consider to enhance `registerTest`, `unregisterTest`, `subscribeTest`, `unsubsribeTest`, etc, instead of introducing new test methods. But your method naming is more recommended, pls. stick to it.